community/lexicon/calendar/event.json

```diff
@@ -0,0 +1,146 @@
+{
+    "lexicon": 1,
+    "id": "community.lexicon.calendar.event",
+    "defs": {
+        "main": {
+            "type": "record",
+            "description": "A calendar event record.",
+            "key": "tid",
+            "record": {
+                "type": "object",
+                "required": [
+                    "createdAt",
+                    "name"
+                ],
+                "properties": {
+                    "name": {
+                        "type": "string",
+                        "description": "The name of the event."
+                    },
+                    "description": {
+                        "type": "string",
+                        "description": "The event description and content for the event."
+                    },
+                    "createdAt": {
+                        "type": "string",
+                        "format": "datetime",
+                        "description": "Client-declared timestamp when this event was originally created."
+                    },
+                    "startsAt": {
+                        "type": "string",
+                        "format": "datetime",
+                        "description": "Client-declared timestamp when this event starts."
+                    },
+                    "endsAt": {
+                        "type": "string",
+                        "format": "datetime",
+                        "description": "Client-declared timestamp when this event ends."
+                    },
+                    "mode": {
+                        "type": "ref",
+                        "ref": "community.lexicon.calendar.event#mode",
+                        "description": "The attendance mode of the event."
+                    },
+                    "status": {
+                        "type": "ref",
+                        "ref": "community.lexicon.calendar.event#status",
+                        "description": "The status of the event."
+                    },
+                    "locations": {
+                        "type": "array",
+                        "description": "The locations where the event takes place.",
+                        "items": {
+                            "type": "union",
+                            "refs": [
+                                "community.lexicon.calendar.event#uri",
+                                "community.lexicon.locations.address",
+                                "community.lexicon.locations.fsq",
+                                "community.lexicon.locations.geo",
+                                "community.lexicon.locations.h3"
+                            ]
+                        }
+                    },
+                    "uris": {
```

Comment:
It's not a required field so not a huge deal, but maybe consider having `embeds` instead of `uris` to allow for eg. link cards in the future?

Reply:
Oh that's a really good idea.

Reply:
I'm missing the terminology,  can someone dumb that down for me?  What's a link card and embeds and how does that differ from a uri?

Reply:
It's probably more that one type of embed is a uri, for example when you post to bsky, links in the middle of the text are extracted and stored separately, so for example you don't have to read "https" in the post content while it does exist in the uri under an embed

This, now merged PR, does not support having embeds if someone, for example, wants to embed a link to their homepage in the middle of the description. It now has to be separately input or is on the atproto dev to extract and do the right thing